frontend/docs/examples.md

```diff
@@ -0,0 +1,31 @@
+---
+id: examples
+title: Examples
```

Comment:
Change this title and the file name to something that depicts what this email is

For example: title can be "Weibenfalk React Slider with Final Space API"

And file name could be react-silder or something. 

P.S. these are just examples.

Reply:
fixed